plugin/backend_checkouts_test.go

```diff
@@ -98,6 +99,10 @@ func ReadReserve(t *testing.T) {
 	if len(serviceAccountNames) != 2 {
 		t.Fatal("expected 2")
 	}
+	disableCheckInEnforcement := resp.Data["disable_check_in_enforcement"].(bool)
+	if !disableCheckInEnforcement {
+		t.Fatal("check-in enforcement should be disabled")
+	}
```

Comment:
Would it be beneficial to add test(s) that check it is enabled, or that users can toggle it back and forth? I'm not sure I have to context to know if this is something that would be changed more than once

Reply:
Sure, I will add a test that toggles it off, then toggles it on again.